apps/box/src/components/Cards/UsersCard.tsx

```diff
@@ -54,15 +50,16 @@ const UserCard = ({ data, ...rest }: UserCardType) => {
         <CardRowData>{new Date(data.connectionDate).toISOString()}</CardRowData>
       </CardRow>
       <FxSpacer marginTop="12" />
-      <FxButton onPress={copyHandler} iconLeft={<CopyIcon />}>
+      <FxButton
+        onPress={() => copyToClipboard(data.decentralizedId)}
```

Comment:
originally I envisioned `copyToClipboard` being a HOF so you could just call `copyToClipboard(data.decentralizedId)` and it would return a callback. i.e onPress could be `onPress={copyToClipboard(data.decentralizedId)}`

But happy with this too